swagger-spec/preprints/definition.yaml

```diff
@@ -45,12 +53,70 @@ properties:
         type: string
         readOnly: false
         description: 'The metadata (copyright year and holder) associated with a license, required for certain licenses.'
-      subjects:
-        type: array
-        items:
-          type: string
+      tags:
+        type: string
+        readOnly: false
+        description: 'The tags given to a preprint.'
+      preprint_doi_created:
+        type: string
+        readOnly: false
+        description: 'When the preprint DOI was created.'
+      date_withdrawn:
+        type: boolean
+        readOnly: false
+        description: 'The date the preprint was withdrawn, null if it was never withdrawn.'
+      current_user_permissions:
+        type: string
+        readOnly: false
+        description: 'The list of permissions the current user has.'
+      public:
+        type: boolean
+        readOnly: false
+        description: 'The wheter the preprint is public.'
+      reviews_state:
+        type: string
+        readOnly: false
+        description: 'The a label to indicate the state of review a preprint is in.'
+      date_last_transitioned:
+        type: string
+        readOnly: false
+        description: 'The date the reviews state of the preprint changed.'
+      has_coi:
+        type: boolean
+        readOnly: false
+        description: 'A value indicating whether the preprint has a conflict of interest statement.'
+      conflict_of_interest_statement:
+        type: string
+        readOnly: false
+        description: 'The text of the conflict of interest statement for the preprint.'
+      has_data_links:
+        type: boolean
+        readOnly: false
+        description: 'A value indicating whether the preprint has a links to data.'
+      why_no_data:
+        type: boolean
+        readOnly: false
+        description: 'A value indicating whether the preprint has a supplemental data.'
```

Comment:
This is a string field for users to explain why a preprint has no data links. Not sure how to word that though